x/avs/keeper/avs.go

```diff
@@ -39,4 +41,21 @@ func (k *Keeper) GetAVSMinimumSelfDelegation(ctx sdk.Context, avsAddr string) (s
 }
 
 // GetEpochEndAVSs returns the AVS list where the current block marks the end of their epoch.
-// func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
+func (k *Keeper) GetEpochEndAVSs(ctx sdk.Context) ([]string, error) {
+	var avsList []types.AVSInfo
+	k.IterateAVSInfo(ctx, func(_ int64, epochEndAVSInfo types.AVSInfo) (stop bool) {
+		avsList = append(avsList, epochEndAVSInfo)
+		return false
+	})
+
+	if len(avsList) == 0 {
+		return nil, nil
```

Comment:
Should return `avsList, nil`?

Reply:
Cannot use 'avsList' (type [] types. AVSInfo) as type [] string